api/src/main/java/org/openmrs/api/ConceptService.java

```diff
@@ -446,6 +449,8 @@
 	 * @param includeRetired include retired concept classes in the search results?
 	 * @throws APIException
 	 * @return List&lt;ConceptClass&gt; object with all ConceptClass objects
+	 * @should return a list of all matching concept classes when include retired is true
+	 * @should return a list of all matching concept classes when include retired is false
 	 */
```

Comment:
same here

* should return all concept classes including retired ones when given true
* should return all concept classes excluding retired ones when given false

(note I removed "matching" since this method returns all without a query parameter)